.github/workflows/docker-build-publish.yml

```diff
@@ -2,11 +2,70 @@ name: Build, Tag, Push, and Release to GitHub Container Registry
 
 on:
   push:
+    branches:
+      - 'master'
+      - 'develop'
+  pull_request:
```

Comment:
Doesn't this change mean that the build and push to docker is going to try run on every single PR?  Since triggers on PRs for all branches?